contrib/utilities/indent

```diff
@@ -71,12 +71,12 @@ process_changed "source" ".*\.inst.in" format_inst
 process_changed "tests include source examples cmake/scripts contrib/python-bindings" \
   ".*\.(cc|h|inst.in|output.*|cmake)" fix_permissions
 
-process_changed "doc/news/*/*" "*" fix_permissions
+process_changed "doc/news/*/*" "[A-Za-z]*" fix_permissions
```

Comment:
The change-log entries start with a number!?

Reply:
Good point. Fixed now.